test/remote/gateways/remote_bambora_apac_test.rb

```diff
@@ -81,6 +81,46 @@ def test_failed_void
     assert_equal 'Cannot find matching transaction to VOID', response.message
   end
 
+  def test_successful_verify
+    response = @gateway.verify(@credit_card, @options)
+    assert_success response
+  end
+
+  def test_failed_verify
+    response = @gateway.verify(credit_card(4123456789010145), @options)
+    assert_failure response
+  end
+
+  def test_successful_store
+    response = @gateway.store(@credit_card, @options)
+    assert_success response
+  end
+
+  def test_failed_store
+    bad_credit_card = credit_card(nil)
+
+    response = @gateway.store(bad_credit_card, @options)
+    assert_failure response
+  end
+
+  def test_successful_purchase_using_stored_card
+    assert store_response = @gateway.store(@credit_card, @options)
+    assert_success store_response
+
+    response = @gateway.purchase(500, store_response.authorization, @options)
+    assert_success response
+    assert_equal '', response.message
```

Comment:
This is a little weird. We usually set the response's `message` to something like "Success".